Update README.md

pmbechard opened the pull request:
Update README.md to improve user experience and accessibility.

pmbechard committed:
Update README.md

Update README.md to improve user experience and accessibility.

sopov commented:
Cool!

Can you return back section with **other questions** or rewrite it as well.

Shortly. I'm not ready to tell someone how to install Golang, download for someone's resume, record a screencast, etc. (if anything changes in the future). I want see in readme that If someone wants some for it, then they can write a manual (like you did)/screencast/etc, if not ready to figure it out themselves, they should go buy a subscription (the price there is not very expensive).

Thanks.

pmbechard committed:
Update README.md

Brings back the Other Questions section with some changes and adds Disclaimer section

pmbechard commented:
I re-added the Other Questions section and reworded it a little. I was also thinking that a Disclaimer might be a good idea here. Let me know what you think.
